vamb/__main__.py

```diff
@@ -1097,6 +1165,20 @@ def load_composition_and_abundance(
     return (composition, abundance)
 
 
+def load_composition_and_abundance_train_only(
+    opt: PartialTrainingOptions,
+) -> Tuple[vamb.parsecontigs.Composition, vamb.parsebam.Abundance]:
```

Comment:
Replace `Tuple` with `tuple`. More pertinent - why do we have this function as separate when training only? Why not use the existing `load_composition_and_abundance`?